diff -up unicode-width-0.1.8/Cargo.toml.1~ unicode-width-0.1.8/Cargo.toml
--- unicode-width-0.1.8/Cargo.toml.1~	2021-03-28 22:39:18.541697041 +0200
+++ unicode-width-0.1.8/Cargo.toml	2021-03-28 22:40:03.242530062 +0200
@@ -14,7 +14,7 @@
 name = "unicode-width"
 version = "0.1.8"
 authors = ["kwantam <kwantam@gmail.com>", "Manish Goregaokar <manishsmail@gmail.com>"]
-exclude = ["target/*", "Cargo.lock"]
+exclude = ["target/*", "Cargo.lock", "/scripts/*"]
 description = "Determine displayed width of `char` and `str` types\naccording to Unicode Standard Annex #11 rules.\n"
 homepage = "https://github.com/unicode-rs/unicode-width"
 documentation = "https://unicode-rs.github.io/unicode-width"
